modules/ROOT/pages/afm-monitoring-private-apis.adoc

```diff
@@ -0,0 +1,53 @@
+
+The following diagram depicts the types of relationship that can exist between monitors, private locations, APIs, and environments. It shows five monitors created in AFM. It also shows a VPC in CloudHub in which there are two associated environments: Environment A and Environment B. In Environment A, there are four private locations and seven APIs being tested. In Environment B, there are one private location and three APIs to be tested. The different line types depict five relationship types.
+
+.The possible relationship types between monitors, schedules, private locations, APIs, and environments
+image::afm-private-locations.png[]
+
+* *First relationship type (represented by solid black arrows):*
++
+Monitor 1 runs on a single schedule. The schedule is associated with Private Location 1. The monitor is written to test one or more endpoints in private API 1 and in private API 2, though it could also test endpoints in more private APIs. The private location and the APIs are all in a single environment.
+
+* *Second relationship type (represented by dashed green arrows):*
++
+Monitor 2 runs on two separate schedules, and each is associated with the same private location. The private location is shared with the schedule that runs Monitor 1. Monitor 2 is written to test one or more endpoints in private API 3 and in private API 4, though it could also test endpoints in more private APIs. The private locations and both APIs are all in a single environment.
+
+* *Third relationship type (represented by dashed-and-dotted blue arrows):*
++
+Monitor 3 runs on two separate schedules, and each is associated with a different private location. Monitor 3 is written to test one or more endpoints in API 5 and in API 6, though it could also test endpoints in more private APIs.
+
+* *Fourth relationship type (represented by dotted orange arrows):*
++
+Monitor 4 runs on two separate schedules, and each is associated with a different private location. Each private location is in a separate environment. One environment could be a test environment, and the other environment could be a production environment. Private API 7 and Private API 8 are copies of the same API. Monitor 4 is written to test one or more endpoints in private API 7 and the same endpoints in API 8, though it could also test endpoints in more private APIs.
+
+* *Fifth relationship type (represented by solid violet arrows):*
++
+Monitor 5 runs on one schedule, though it could run on more than one. The schedule is associated with a private location. Monitor 5 is written to test one or more endpoints in private API 9 and in public API 10.
+
+== Length of Time Before Tests Time Out
+
+Because tests are run in a VPC, the workers do not share resources with other MuleSoft customers. Therefore, AFM allows tests to wait for a default of thirty minutes for a response from an endpoint. You can increase this time in Runtime Manager. To do so, follow these steps:
```

Comment:
```suggestion
Because tests are run in an Anypoint VPC, the workers do not share resources with other MuleSoft customers. Therefore, API Functional Monitoring allows tests to wait for a default of thirty minutes for a response from an endpoint. You can increase this time in Anypoint Runtime Manager. To do so, follow these steps:
```

Reply:
Resolved.